### DISCUSSION TITLE
Add milliwatt-hour as unit of energy

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/architecture#1175] [Entity Models] [answered]
Home Assistant currently has Wh (watt-hour), kWh (kilowatt hour) MWh (megawatt-hour) GWh (gigawatt-hour) and TWh (terawatt-hour) as units of power for electrical energy.

Since Matter v1.3, home appliances using the protocol can report their energy consumption using the Electrical Energy Measurement clusters. The energy measurement cluster reports energy consumed in mWh (milliwatt-hour).

## Proposal

Add the milliwatt hour equivalent as unit of energy measurement:

- `mWh` (milliwatt-hour) as unit of energy

Implementation for [mWh as unit of energy](https://github.com/home-assistant/core/pull/133005) is already present.

## Integrations which can benefit from the proposal

Integrations that can benefit are primarily for devices with accurate electrical energy measurement:

- Matter

**Comment by parautenbach** [comment]
Watt is already a rate (i.e. J/s or Joule per second). Wh, etc., is a unit of energy (J or Joule). Wh is _not_ W/h – it's literally W multiplied by h, so Watt-hour.

**Reply by emontnemery** [reply]
Corrected

**Reply by agners** [reply]
Right, thanks for the correction!

**Comment by frenck** [comment, marked as answer]
This was approved and added ✅
